sections/collapsible-content.liquid

```diff
@@ -0,0 +1,383 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content collapsible-{{ section.settings.layout | split: '_' | first }}-layout color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content__wrapper{% if section.settings.layout == 'section_container' %} color-{{ section.settings.container_color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
```

Comment:
`collapsible-content-wrapper` doesn't seem to exist so it doesn't make sense to have `--narrow` modifier
```suggestion
    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper-narrow{% else %} page-width{% endif%}">
```